contrib/capitalone_dataprofiler_expectations/README.md

```diff
@@ -290,7 +290,7 @@ The format for an unstructured profile is below:
 * CREDIT_CARD
 * EMAIL_ADDRESS
 * UUID
-* HASH_OR_KEY (md5, sha1, sha256, random hash, etc.)
```

Comment:
I would leave this as is and revert this change.
Data Profilers are a legacy part of our codebase. Users can elect to use this or not and can elect which features they want. A FIPS user would not use the md5 but other users might find it useful for their datasets.